arceos-vmm/scripts/make/bsta1000b-fada.mk

```diff
@@ -1,4 +1,4 @@
 fada: build
 	gzip -9 -cvf $(OUT_BIN) > arceos-fada.bin.gz
-	mkimage -f tools/bsta1000b/bsta1000b-fada-arceos.its arceos-fada.itb
+	mkimage -f configs/bsta1000b-fada-arceos.its arceos-fada.itb
```

Comment:
just leave it in tools or create a seperate repo.

do not include them in `configs/` dir, it's already too complicated.

Reply:
ok

Reply:
now the tools will be downloaded from remote server